prov/verbs/src/verbs_eq.c

```diff
@@ -349,6 +349,7 @@ static int fi_ibv_eq_close(fid_t fid)
 	}
 
 	dlistfd_head_free(&eq->list_head);
+	fastlock_release(&eq->lock);
```

Comment:
Why is this function grabbing the lock at all?  The entire structure and memory are about to be freed, so the lock can't protect against anything.